internal/pkg/service/buffer/statistics/cache/cache.go

```diff
@@ -0,0 +1,205 @@
+// Package cache provides local cache for files and slices statistics.
+// It is used for fast resolving of the upload/import conditions.
+// The cache is synced via the etcd Watch API.
+package cache
```

Comment:
Nakoniec som namiesto `statistics/workernode` zvolil `statistics/cache` ... kedze sa to moze neskor pouzit aj vramci API, ak by bol nejaky endpoint na nacitanie statistik.